inductiva/stellarators/scenarios/coils/coils.py

```diff
@@ -9,13 +9,13 @@
 
 import numpy as np
 
-import inductiva
+from inductiva import scenarios, simulation, stellarators, tasks
```

Comment:
You should be able to do everything with inductiva. I think only keep `stellerators.simulators.Simsopt` may be more difficult, but if you can pull it off it will be preferable.

Reply:
I changed it because I needed to add `from . import stellarators` in the inductiva init file. And so if in here I just did `import inductiva` it did not work because it was a circular import.